src/com/gfredericks/test/chuck/clojure_test/impl.cljc

```diff
@@ -0,0 +1,29 @@
+(ns com.gfredericks.test.chuck.clojure-test.impl
+  (:require #?(:clj  [clojure.test :refer [is]]
+               :cljs [cljs.test :refer-macros [is]])))
+
+;; copied from clojure.test.check, which privatized the function in
+;; recent versions.
+;;
+;; I think there might be plans for test.check to abstract this logic
+;; into a protocol or something, so I'm not too bothered by the
+;; copypasta for now.
+(defn ^:private not-falsey-or-exception?
+  [value]
+  (and value (not (instance? #?(:clj  Throwable
+                                :cljs js/Error)
```

Comment:
oh this is interesting -- js let's you throw arbitrary objects, doesn't it? Does that mean that we can't currently distinguish a thrown not-`js/Error`-object from a returned object?

There are some changes in mind to test.check that would make this a bit cleaner, and I'm okay living with this for now, I just wanted to check if that's indeed the case.